proxy/src/coco/source.rs

```diff
@@ -237,6 +242,18 @@ impl TryFrom<Revision> for Rev {
     }
 }
 
+/// Bundled response to retrieve both branches and tags for a user repo.
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct UserRevisions {
+    /// Owner of the repo.
+    pub(crate) identity: identity::Identity,
```

Comment:
Just curious - why are these only visible in the `crate`?

Reply:
Is this a best practice we should be following elsewhere too?

Reply:
They were private, so I went one level up. I suppose since `proxy` is actually the only consumer of this library it could just be `pub`.

What's your stance on this @xla?